### PR TITLE
[WOR-646] Update swagger page

### PR DESCRIPTION
**Pull request by ghost**
Ticket: https://broadworkbench.atlassian.net/browse/WOR-646
Related to [Add cloudPlatform to workspace details endpoint ](https://github.com/broadinstitute/rawls/pull/2151)

Have you read [CONTRIBUTING.md](../CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've followed [the instructions](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#api-changes) if I've made any changes to the API, _especially_ if they're breaking changes
- [ ] I've updated the RC_XXX release ticket with any manual steps required to release this change
- [ ] I've updated the [FISMA documentation](https://github.com/broadinstitute/firecloud-orchestration/blob/develop/CONTRIBUTING.md#fisma-documentation-changes) if I've made any security-related changes, including auth, encryption, or auditing

In all cases:

- [ ] Get two thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this **unless it's for a hotfix**. In that case, don't delete it!
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by cahrens — APPROVED** [review]

File: src/main/resources/swagger/api-docs.yaml [hunk 1]
```diff
@@ -9255,6 +9255,9 @@ components:
           description: The date all files were successfully transferred to the workspace bucket if applicable
             in yyyy-MM-ddTHH:mm:ss.SSSZZ format
           format: date-time
+        cloudPlatform:
+          type: string
+          description: The cloud platform of the workspace
```
Comment (cahrens) [inline src/main/resources/swagger/api-docs.yaml:9260]: Actually, wondering if we should add the possible values, like we do for the billing project response.

cloudPlatform:
          type: string
          enum:
            - GCP
            - AZURE

Comment (cahrens) [reply src/main/resources/swagger/api-docs.yaml:9260]: And should this be listed under "required"?

**Review by cahrens — CHANGES_REQUESTED** [review]
Had a few comments/suggestions. These will likely require some cleanup in Rawls too.

File: src/main/resources/swagger/api-docs.yaml (hunk 1 above)
Comment (marctalbott) [reply src/main/resources/swagger/api-docs.yaml:9260]: It should not be required as the cloud platform is optional on the underlying `WorkspaceDetails` case class in Rawls and may not be present in the response

**Review by marctalbott — APPROVED** [review]

**Review by cahrens — APPROVED** [review]

**Pull request merged**